From b48831493437eda771e44fb94794db3582efd54c Mon Sep 17 00:00:00 2001
From: Louis Bouchard <louis.bouchard@canonical.com>
Date: Thu, 28 Aug 2014 10:06:31 +0200
Subject: fix ceilometer stack

Description: fix ceilometer stack

Remove error in execution of the ceilometer plugin

Author: Louis Bouchard <louis.bouchard@canonical.com>
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1355279

Signed-off-by: Louis Bouchard <louis.bouchard@canonical.com>
---
 sos/plugins/openstack_ceilometer.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/sos/plugins/openstack_ceilometer.py b/sos/plugins/openstack_ceilometer.py
index 524b68b..7cd9091 100644
--- a/sos/plugins/openstack_ceilometer.py
+++ b/sos/plugins/openstack_ceilometer.py
@@ -27,7 +27,10 @@ class OpenStackCeilometer(plugins.Plugin):
 
     def setup(self):
         # Ceilometer
-        self.add_copy_spec("/etc/ceilometer/", "/var/log/ceilometer")
+        self.add_copy_specs([
+            "/etc/ceilometer/",
+            "/var/log/ceilometer"
+        ])
 
 class DebianOpenStackCeilometer(OpenStackCeilometer, plugins.DebianPlugin, plugins.UbuntuPlugin):
     """OpenStackCeilometer related information for Debian based distributions."""
